android/src/main/java/com/tadasr/IOTWifi/IOTWifiModule.java

```diff
@@ -13,47 +16,104 @@
 
 import java.util.List;
 
+
+class FailureCodes {
+    static int SYSTEM_ADDED_CONFIG_EXISTS = 1;
+    static int FAILED_TO_CONNECT = 2;
+    static int FAILED_TO_ADD_CONFIG = 3;
+    static int FAILED_TO_BIND_CONFIG = 4;
+}
+
 public class IOTWifiModule extends ReactContextBaseJavaModule {
-    WifiManager wifiManager;
-    ConnectivityManager connectivityManager;
-    ReactApplicationContext context;
-    
+    private WifiManager wifiManager;
+    private ConnectivityManager connectivityManager;
+    private ReactApplicationContext context;
+
     public IOTWifiModule(ReactApplicationContext reactContext) {
         super(reactContext);
-        wifiManager = (WifiManager) getReactApplicationContext().getApplicationContext().getSystemService(Context.WIFI_SERVICE);
-        connectivityManager = (ConnectivityManager) getReactApplicationContext().getApplicationContext().getSystemService(Context.CONNECTIVITY_SERVICE);
+        wifiManager = (WifiManager) getReactApplicationContext().getApplicationContext()
+                .getSystemService(Context.WIFI_SERVICE);
+        connectivityManager = (ConnectivityManager) getReactApplicationContext().getApplicationContext()
+                .getSystemService(Context.CONNECTIVITY_SERVICE);
         context = getReactApplicationContext();
     }
-    
+
+    private String errorFromCode(int errorCode) {
+        return "Error: " + errorCode;
```

Comment:
I should have used `ErrorCode`, can you update it please?

Reply:
I assume you mean it should just be capitalized?

Reply:
I meant
`"Error: " + errorCode`; 
should be 
`"ErrorCode: " + errorCode;`

Reply:
errorCode is an argument to the java method so it shouldn't be capitalized. Sorry about the confusion.

Reply:
Got it I will change that around. i thought the request was odd.